src/data/repositories/ASTGuidelinesD2Repository.ts

```diff
@@ -0,0 +1,51 @@
+import { D2Api } from "@eyeseetea/d2-api/2.36";
+import { DataStoreClient } from "../DataStoreClient";
+import { ASTGuidelinesRepository } from "../../domain/repositories/ASTGuidelinesRepository";
+import { ASTGuidelinesContextState } from "../../webapp/contexts/ast-guidelines-context";
+import { FutureData } from "../api-futures";
+import { DataStoreKeys } from "../DataStoreKeys";
+
+export class ASTGuidelinesD2Repository implements ASTGuidelinesRepository {
+    constructor(private dataStoreClient: DataStoreClient, private api: D2Api) {}
```

Comment:
D2Api not used fir now, but maybe it will be in the future?

Reply:
Removed.